app/src/ui/open-pull-request/open-pull-request-dialog.tsx

```diff
@@ -208,6 +208,7 @@ export class OpenPullRequestDialog extends React.Component<IOpenPullRequestDialo
         className="open-pull-request"
         onSubmit={this.onCreatePullRequest}
         onDismissed={this.props.onDismissed}
+        width={{ value: 900, max: 1100, min: 850 }}
```

Comment:
> max: 1100

Arbitrary... but no idea what we would make since as a maximum.

> min: 850

Not sure about this either.. 850 was chosen based on min window with being 960 - 100 for some margins. However, I am inclined to put this as -infinity as the logic should bear this out seeing as the resize function won't be called for less the the app width.